client/blocks/product-purchase-features/product-purchase-features-list/custom-domain.jsx

```diff
@@ -2,19 +2,17 @@
  * External dependencies
  */
 import React from 'react';
+import pick from 'lodash/pick';
 
 /**
  * Internal dependencies
  */
 import CustomDomainPurchaseDetail from 'my-sites/upgrades/checkout-thank-you/custom-domain-purchase-detail';
 
-export default ( { selectedSite, hasDomainCredit } ) => {
+export default ( props ) => {
 	return (
 		<div className="product-purchase-features-list__item">
-			<CustomDomainPurchaseDetail
-				selectedSite={ selectedSite }
-				hasDomainCredit={ hasDomainCredit }
-			/>
+			<CustomDomainPurchaseDetail { ...pick( props, [ 'selectedSite', 'hasDomainCredit', 'isPressableSite' ] ) } />
```

Comment:
Me too. I think we should pass all the props (with picking/omitting some) only if they are passed to a native component like `div` or `button`. Those component can accept various different props so it's better to simply pass down everything instead of listing the possible props.

However, in this case, we are passing down only three props so it's unnecessary to use this pattern IMO. :)